templates/barbican/config/barbican-api-config.json

```diff
@@ -77,6 +77,14 @@
       "perm": "0600",
       "optional": true,
       "merge": true
+    },
+    {
+      "source": "/var/lib/config-data/default/Crystoki.conf",
+      "dest": "{{ .HSMLibraryPath }}/config/Crystoki.conf",
+      "owner": "barbican",
+      "perm": "0600",
+      "optional": true,
```

Comment:
This will work for now until we learn how to do this properly, but the way things are right now - this will always be added.  why?  because Crystoki.conf is in templates/barbican/config/ and therefore will always be templated - even if we're not using  a luna